test/asm_test.rb

```diff
@@ -168,7 +158,7 @@ def test_make_elf
   # this test can be removed after method +run_shellcode+ being implemented
   def test_make_elf_and_run
     # run the ELF we created to make sure it works.
-    linux_only
+    linux_only('ELF can only be executed on Linux')
```

Comment:
No ()?
(There's one linux_only without () below)
```suggestion
    linux_only 'ELF can only be executed on Linux'
```

Reply:
wow the feature of `suggested change` is great